scripts/variantstore/AOU_DELIVERABLES.md

```diff
@@ -9,37 +9,38 @@
     	- Make sure that any samples that have been withdrawn or that have been loaded but should not be in the callset have the is_loaded column set to false.
     3. GvsCreateAltAllele
     4. GvsCreateFilterSet
-        - make note of the “filter\_set_name” input used
+        - make note of the "filter_set_name" input used
     5. GvsPrepareCallset
-        - if the ```sample_info``` table contains control samples, create a view in your dataset called ```sample_info_without_controls``` populated bby the query ```SELECT * FROM `aou-genomics-curation-prod.beta_release.sample_info` where sample_id > 50```.
-        - if the callset has more than 20K genomes, you will want to set the attribute skip_vet_new_insert to true which will run all of the queries except the last one. It will print out the last query that needs to run with flex slots. Allocate the flex slots, run the query, and then unallocate them. 
+        - If the ```sample_info``` table contains control samples, create views in your dataset called ```sample_info_without_controls``` populated by the query ```SELECT * FROM `aou-genomics-curation-prod.beta_release.sample_info` where sample_id > 50``` and ```sample_info_controls_only``` populated by the query ```SELECT * FROM `aou-genomics-curation-prod.beta_release.sample_info` where sample_id <= 50```. A run for ```sample_info_without_controls``` will create the tables for the joint callset deliverable.  The run with ```sample_info_controls_only```  (with a distinct `destination_cohort_table_prefix` from the non-control run) will be used for the ["Sensitivity and Precision" deliverable](tieout/AoU_PRECISION_SENSITIVIT.md).
+        - If the callset has more than 20K genomes, you will want to set the input "skip_vet_new_insert" to `true` which will run all of the queries except the last one. It will print out the last query that needs to run with flex slots; allocate the flex slots, run the query, and then deallocate them.
```

Comment:
are we going to give more in depth instructions of allocating and deallocating flex slots somewhere else? or is it assumed that people will know how to do this?

Reply:
Since the prepare step is going away, it's probably not worth it to go into flex slots right now.